wp-postratings.php

```diff
@@ -1163,7 +1163,11 @@ function expand_ratings_template($template, $post_data, $post_ratings_data = nul
 
 	// Post Template Variables
 	$post_link = get_permalink( $post_data );
-	$post_title = get_the_title( $post_data );
+	if ((get_post_meta($post->ID, ‘_yoast_wpseo_title’, true)) == “”) {
```

Comment:
Your single and double quotes are wrong.
Why not use `$post_id`?

`if ( ( get_post_meta( $post_id, '_yoast_wpseo_title', true ) ) === '' ) {`

Reply:
You're correct.
I did the git on a tablet that mucked the quotes.
Also, moved to post_id as suggested.